spec/rails_helper.rb

```diff
@@ -74,10 +74,10 @@
 end
 
 VCR.configure do |config|
+  # config.allow_http_connections_when_no_cassette = true
   config.cassette_library_dir = "spec/fixtures/vcr_cassettes"
   config.hook_into :webmock
-  config.filter_sensitive_data('<api_key>') { ENV['API_KEY'] }
+  config.filter_sensitive_data('<api_key>') { ENV['cocktail_key'] }
   config.default_cassette_options = { re_record_interval: 4.days }
```

Comment:
So, is this recording a new cassette every 4 days? Curious about how that is working.